app/content/models/notification.py

```diff
@@ -33,3 +32,7 @@ def has_object_write_permission(self, request):
         if request.method == "PUT":
             return self.user == request.user
         return False
+
+
+def notification_creator(user, message):
```

Comment:
Hvis vi skal følge `action_object()` konvensjonen foreslår jeg følgende:
```suggestion
def create_notification(user, message):
```